site/en/docs/privacy-sandbox/chips/index.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'CHIPS—Cookies Having Independent Partitioned State'
+subhead: >
+  Allow developers to opt-in a cookie to "partitioned" storage, with a separate cookie jar per top-level site.
+description: >
+  CHIPS is a Privacy Sandbox proposal that will allow cookies to be set by a third-party service, but only read within the context of the top-level site where they were initially set
+date: 2022-02-15
+authors:
+  - mihajlija
+---
+
+## What is CHIPS
+
+Cookies Having Independent Partitioned State (CHIPS) is a Privacy Sandbox proposal that will allow developers to opt-in a cookie to "partitioned" storage, with a separate cookie jar per top-level site.
+
+The aim is to allow cookies to be set by a third-party service, but only read within the context of the top-level site where they were initially set. A partitioned third-party cookie is tied to the top-level site where it was initially set and cannot be accessed from elsewhere.
+
+### Why do we need it
+
+Currently, third-party cookies can enable services to track users and join their information from across many unrelated top-level sites. This is known as cross-site tracking. 
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/jLOlPtLY9Zqte4IOoU6g.png", alt="ALT_TEXT_HERE", width="800", height="450" %}
+
+For example, when a user visits site A, embedded content from site C can set a cookie on the user's machine in response to the cross-site request. If the user then visits site B which also embeds C, site C could access the same cookie that has been set earlier when the user visited site A. This allows site C to compile a user's browsing activity across site A, site B, and every site that it is embedded on.
+
+To protect user privacy, browser vendors are placing restrictions on this behavior and phasing out support for third-party cookies.
+
+While cross-site tracking is an issue, there are still valid cross-site cookie use cases on the web today which can be achieved in a privacy-preserving way with cookie partitioning.
+
+### Use cases
+
+For example, the site `retail.example` may want to work with a third-party service `support.chat.example` to embed a support chat box on its site. Many embeddable chat services today rely on cookies to save interaction history. 
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/jsxgCpkMRXwXughPjg7j.png", alt="ALT_TEXT_HERE", width="800", height="620" %}
+
+Without the ability to set a cross-site cookie, `support.chat.example` could instead rely on `retail.example` passing along their first-party session identifier (or some derived value of it). In that case, every website that `support.chat.example` is embedded on would require additional setup to pass along the state.  
```

Comment:
I'm actually not sure what state is -- and maybe that's my lack of knowledge of cookies. 

I reviewed the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies). It's possible I'm overthinking it with my lack of knowledge, but it may be helpful to define. Such as:

> In that case, additional setup would be required to pass along session information and personalization (otherwise known as the _state_) for every website that embeds `support.chat.example`.